### ISSUE TITLE
File manager small UI issues

### ISSUE THREAD

**Issue by lunarovic** [authentic-theme/authentic-theme#1240]
1. Vertical and horizontal scrollbar are hard to select with the mouse, because only half of the pixels will select the scrollbar and the other half (closer to the edge) will activate the resizing cursor. I often resize the window by accident when I wanted to scroll it.

2. When you right-click on a file or folder and click (accidentally) on a divider line in the context menu, the menu will close. In Windows UI it doesn't close.

**Comment by lunarovic** [comment]
![image](https://user-images.githubusercontent.com/2381553/48673927-042c0800-eb47-11e8-8525-9c9d4f81f129.png)
The cursor turns to resize and you can't click on the right half of the scrollbar. Same for the horizontal scrollbar.

**Comment by rostovtsev** [comment]
That is odd, as I have it displayed as pointer.

![screenshot from 2018-11-18 17-48-14](https://user-images.githubusercontent.com/4426533/48674162-46ab1000-eb5a-11e8-8929-877ff9bc6e95.png)


What browser, OS are you using?

**Comment by lunarovic** [comment]
I'm sorry, I forgot to state that I refer to scrollbars in the **file editor**, my mistake. Those scrollbars in the tree-view panel are OK on my side too. I'm using Chrome 70 on Windows 10.

---

By the way, one really small thing I found, look at how the question mark icon overlaps text if the width of the screen is small:
![image](https://user-images.githubusercontent.com/2381553/48674795-7a813800-eb50-11e8-9fbc-7c6d3e5b469d.png)

When I hover, we can see the white rectangle better:
![image](https://user-images.githubusercontent.com/2381553/48674806-9dabe780-eb50-11e8-9a26-4d33b257b38d.png)

This is from the File manager module config by the way, I went to turn on the tree-view panel, because I don't use it. :)
